outport/process/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -242,10 +249,14 @@ func (aap *alteredAccountsProvider) addAddressWithBalanceChangeInMap(
 
 	_, addressAlreadySelected := markedAlteredAccounts[string(address)]
 	if addressAlreadySelected {
+		markedAlteredAccounts[string(address)].isSender = markedAlteredAccounts[string(address)].isSender || isSender
```

Comment:
Please extract `string(address)` in a temp variable `addressStr` and also use it below on `L256`

Reply:
done